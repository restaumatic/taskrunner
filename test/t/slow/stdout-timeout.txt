# By spawning a background process, we add to refcount of the pipe.
sleep 2 &
disown
