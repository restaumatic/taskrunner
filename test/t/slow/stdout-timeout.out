-- output:
[toplevel] error | Task did not close stdout 1 seconds after exiting.
[toplevel] error | Perhaps there's a background process?
-- exit code: 1
